es_ES/_data/catalog/widgets.json

```diff
@@ -0,0 +1,1582 @@
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "RawImage",
+    "description": "Un widget que muestra una dart:ui.Image directamente.",
+    "categories": [
+      "Assets, Images, and Icons"
+    ],
+    "subcategories": [
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/RawImage-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "AssetBundle",
+    "description": "Los paquetes de activos contienen recursos, como imágenes y cadenas, que pueden ser utilizados por una aplicación. El acceso a estos recursos es asincrónico, por lo que se pueden cargar de forma transparente a través de una red (por ejemplo, desde un NetworkAssetBundle) o desde el sistema de archivos local sin bloquear la interfaz de usuario de la aplicación.",
+    "categories": [
+      "Assets, Images, and Icons"
+    ],
+    "subcategories": [
+    ],
+    "link": "https://docs.flutter.io/flutter/services/AssetBundle-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "Form",
+    "description": "Un contenedor opcional para agrupar varios widgets de campo de formulario (por ejemplo, widgets TextField).",
+    "categories": [
+      "Input"
+    ],
+    "subcategories": [
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/Form-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "FormField",
+    "description": "Un solo campo de formulario. Este widget mantiene el estado actual del campo de formulario, por lo que las actualizaciones y los errores de validación se reflejan visualmente en la interfaz de usuario.",
+    "categories": [
+      "Input"
+    ],
+    "subcategories": [
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/FormField-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "RawKeyboardListener",
+    "description": "Un widget que realiza llamadas de regreso (callbacks) cuando sea que el usuario presiona o suelta una tecla en el teclado.",
+    "categories": [
+      "Input"
+    ],
+    "subcategories": [
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/RawKeyboardListener-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "AnimatedContainer",
+    "description": "Un contenedor que gradualmente cambiar su valor en un período de tiempo.",
+    "categories": [
+      "Animation and Motion"
+    ],
+    "subcategories": [
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/AnimatedContainer-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "AnimatedCrossFade",
+    "description": "Un widget que hace desvanecimientos cruzados entre dos hijos dados y realiza animaciones en si mismo entre sus tamaños.",
```

Comment:
A widget that cross-fades between two given children and animates itself between their sizes.
Un widget que realiza una animación cross-fade entre dos hijos dados, realizando la transición entre sus dimensiones.